src/renderers/stats.ts

```diff
@@ -479,9 +481,14 @@ function generateStoresChart (storeSums: Map<string, StoreSum>): HTMLDivElement
   return div
 }
 
+let addedChartsCss = false
```

Comment:
Whoops. This bit is a remnant from before I implemented `addedCache` in src/lib/css.ts. Will remove.